tensorflow/python/keras/backend.py

```diff
@@ -2492,6 +2492,17 @@ def concatenate(tensors, axis=-1):
 
   Returns:
       A tensor.
+
+  Example:
+      ```python
+      >>> a = tf.constant([[1, 2, 3], [4, 5, 6], [7, 8, 9]])
+      >>> b = tf.constant([[10, 20, 30], [40, 50, 60], [70, 80, 90]])
+      >>> tf.keras.backend.concatenate((a, b), axis=1)
```

Comment:
say -1 even though it's the same in this case, since that's the default arg value.

Reply:
Thanks, 56d39f9 brings the change.